### ISSUE TITLE
About the domain of these call graphs

### ISSUE THREAD

**Issue by zhaohan-xi** [quarkslab/dataset-call-graph-blogpost-material#1]
Hi Dear Author, this call graph dataset is excellent and could be widely used as a domain-specific benchmark, and I see the categories are goodware and malware. Since there are a lot of malware: Android Malware, Network Malware, or other types pf system malware, which type this dataset belongs to? And from what domain of platforms these call graphs were collected?

**Comment by Moulourf** [comment]
Hi,

"this call graph dataset is excellent and could be widely used as a domain-specific benchmark"

It was the initial goal but the associated academic paper never came to fruition...

"Since there are a lot of malware: Android Malware, Network Malware, or other types pf system malware, which type this dataset belongs to?"

We did not focus on a particular type of malware but only on malicious PE32, therefore many malware types are represented in this dataset.
A classification of these callgraphs into different categories  could be done from the initial dataset (consisting of PE).

"And from what domain of platforms these call graphs were collected?"

If my memory is correct, the malware call graphs were extracted from virus share and the goodware call graphs were extracted from the binaries present on a fresh installation of windows 7 as well as from binaries downloaded from sourceforce (https://sourceforge.net/, with a constraint on the popularity).

VirusShare is an academic database of malicious files collected via several honeypots (https://virusshare.com/).
I used a fresh torrent at the time in order to obtain zero days that were bypassing standard antivirus.

I point out that obtaining a more actual and representative dataset is possible and could easily be done (scrapping and then use IDA or Radare2 to generate call graphs).

If you have any further questions or require further information I can be reached directly at leo.rannou@gmail.com

VirusShare.com<https://virusshare.com/>
MD5: 1705351ffd14a23bd275ec53c633e204: SHA1: 28402bafb9df1a0e8296ddfc33edb200a74f4314: SHA256: f6f2d71e9895534870bf30eec519e2f2c4838d276907bfa05909d8bf06f6d411
virusshare.com


________________________________
De : HarrialX <notifications@github.com>
Envoyé : jeudi 10 décembre 2020 01:16
À : quarkslab/dataset-call-graph-blogpost-material <dataset-call-graph-blogpost-material@noreply.github.com>
Cc : Subscribed <subscribed@noreply.github.com>
Objet : [quarkslab/dataset-call-graph-blogpost-material] About the domain of these call graphs (#1)


Hi Dear Author, this call graph dataset is excellent and could be widely used as a domain-specific benchmark, and I see the categories are goodware and malware. Since there are a lot of malware: Android Malware, Network Malware, or other types pf system malware, which type this dataset belongs to? And from what domain of platforms these call graphs were collected?
